src/app/content/components/Wrapper.tsx

```diff
@@ -1,9 +1,24 @@
-import styled from 'styled-components/macro';
+import { connect } from 'react-redux';
+import styled, { css } from 'styled-components/macro';
 import { LayoutBody } from '../../components/Layout';
-
+import { AppState } from '../../types';
+import * as selectSearch from '../search/selectors';
 export { wrapperPadding } from '../../components/Layout';
 
-export default styled(LayoutBody)`
+// tslint:disable-next-line:variable-name
+export const Wrapper = styled(LayoutBody)`
   position: relative; /* for sidebar overlay */
   overflow: visible; /* so sidebar position: sticky works */
+  flex: 1;
+  ${(props) => {
+    if (props.open) {
```

Comment:
this could use a more descriptive name like `searchOpen` or something, and can be reformatted like

```
styled(asdf)`
  ${(props) => props.open && css`
    display: none;
  `}
`;